cmd/daily/update/update.go

```diff
@@ -0,0 +1,26 @@
+package update
+
+import (
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update a daily file",
+		Long:  "Update a daily file in your cronicle journal.",
+		Run:   run,
+	}
+
+	cmd.Flags().IntP("number", "n", 0, "number on ordered list to update")
+	cmd.MarkFlagRequired("number")
+	cmd.Flags().StringP("message", "m", "", "content of your todo")
```

Comment:
`message` should be required, right?

Reply:
For update, maybe we just open the file in `vim` for the user? Something like this: https://stackoverflow.com/questions/21513321/how-to-start-vim-from-go